Swiftfin tvOS/Views/VideoPlayer/Overlays/SmallMenuOverlay.swift

```diff
@@ -170,5 +189,22 @@ extension VideoPlayer {
                 .focused(focused, equals: section)
             }
         }
+
+        struct MenuStyle: ViewModifier {
+            var focusGuide: FocusGuide
+
+            func body(content: Content) -> some View {
+                content
+                    .frame(height: 80)
+                    .padding(.horizontal, 50)
+                    .padding(.top)
+                    .padding(.bottom, 45)
+                    .focusGuide(
```

Comment:
This ordering causes the spacing issue. The `focusGuide` should come before the `frame(height: 80)`.

Reply:
Nice, this looks a lot better now.

![simulator_screenshot_1544093D-2A3C-41CB-8AF5-9368A75ACC57](https://github.com/jellyfin/Swiftfin/assets/22553678/c9bd08a1-d65c-47f2-b405-55764a2a7518)